### ISSUE TITLE
[Question] 1.19.3

### ISSUE THREAD

**Issue by Aces-and-Jacks** [wisp-forest/CondensedCreativeItems#4]
With the inventory being reorganized and certain items not next to each other would that mean you wouldn't be able to do them? 

For example the buttons used to be next to each other but from what I've noticed they are instead separated by wood category with their wood types instead of being right next to each other. However in the colored blocks tab those items are next to each other, like grouped by the item type (red candle, white candle, ect). 

So basically what I'm asking is are some items going to not be condensed since they aren't next to each other. If you end decide to update the mod

**Comment by Dragon-Seeker** [comment]
> With the inventory being reorganized and certain items not next to each other would that mean you wouldn't be able to do them?

I will most likely handle it like previously and have the entry generated where ever the first item is found. The system in place will attempt to remove any item found with the given entry and currently keep the order at which the entry was built with.
